crates/bindings-sys/src/lib.rs

```diff
@@ -745,3 +747,155 @@ impl Drop for RowIter {
         unsafe { raw::_iter_drop(self.raw) }
     }
 }
+
+/// Rust tries to build tests for our various modules on the host system, not for wasm.
+/// On Linux, this succeeds, despite there being unresolved symbols, because the linker is okay with
+/// unresolved symbols.
+/// On Windows, this fails, because the linker errors on unresolved symbols.
+/// By adding these shims, we can build and run tests on windows, as long as they don't use module ABI calls.
+#[cfg(target_family = "windows")]
```

Comment:
This DOES mean that if you, for some reason, wanted to run modules in native code, it would be forbidden on windows.
I thought about alternatively doing `#[cfg(not(target_family = "wasm"))]` so that it would be forbidden everywhere non-wasm instead. Not sure what the right choice is.